src/constants/papersDefinitions.json

```diff
@@ -5,12 +5,22 @@
       "icon": "people",
       "featuredPlaceholder": true,
       "rankPlaceholder": 1,
-      "pagesNumber": 2,
+      "pagesNumber": 4,
```

Comment:
I don't know if we should store this information. Since we can have it by checking the `pages.length`. Avoiding data duplication is good 👍

Reply:
Effectivement cette donnée n'est plus utile je pense, elle avait été mise à la base pour éviter de répéter les objets dans `pages`, pour les rares cas où ça serait les mêmes, mais ce n'est plus pertinent aujourd'hui.